gluten-core/src/main/scala/io/glutenproject/execution/GenerateExecTransformer.scala

```diff
@@ -57,8 +57,14 @@ case class GenerateExecTransformer(
   extends UnaryExecNode
   with TransformSupport {
 
+  @transient
+  override lazy val metrics =
+    BackendsApiManager.getMetricsApiInstance.genGenerateTransformerMetrics(sparkContext)
+
   override def output: Seq[Attribute] = requiredChildOutput ++ generatorOutput
 
+  override def producedAttributes: AttributeSet = AttributeSet(generatorOutput)
```

Comment:
this is a small bug fix, to fix the missing input issue. the `!` of GenerateExecTransformer

`^(15) !GenerateExecTransformer explode(array(1, c1#0)), false, [col#74]`